binder/postBuild

```diff
@@ -31,7 +31,7 @@ chmod +x xstartup
 # Desktop applications
 # Fiji and OMERO insight
 
-OMERO_INSIGHT_VERSION=5.5.11
+OMERO_INSIGHT_VERSION=5.5.14
 OMERO_INSIGHT_URL_BASE=https://github.com/ome/omero-insight/releases/download/v$OMERO_INSIGHT_VERSION
```

Comment:
Linkchecker is unhappy. Indeed, when I constructed the url as indicated here, this is

```
https://github.com/ome/omero-insight/releases/download/v5.5.14
```
This url is giving 404

Reply:
The URL to check is the one of https://github.com/ome/omero-guide-fiji/blob/master/binder/postBuild#L43 -
```
OMERO_INSIGHT_VERSION=5.5.14
OMERO_INSIGHT_URL_BASE=https://github.com/ome/omero-insight/releases/download/v$OMERO_INSIGHT_VERSION
wget -q $OMERO_INSIGHT_URL_BASE/omero_ij-$OMERO_INSIGHT_VERSION-all.jar
```
works just fine.

I don't see that the linkchecker reports are related to this PR. Reading the travis log, it successfully does,
```
wget -q https://github.com/ome/omero-insight/releases/download/v5.5.14/omero_ij-5.5.14-all.jar
```